src/main/java/net/dv8tion/jda/api/requests/restaction/interactions/PremiumRequiredCallbackAction.java

```diff
@@ -0,0 +1,13 @@
+package net.dv8tion.jda.api.requests.restaction.interactions;
+
+import net.dv8tion.jda.api.requests.FluentRestAction;
+
+/**
+ * An {@link InteractionCallbackAction} that can be used to send the "Premium required" interaction response.
+ *
+ * @see    net.dv8tion.jda.api.interactions.callbacks.IPremiumRequiredReplyCallback
+ */
+public interface PremiumRequiredCallbackAction extends InteractionCallbackAction<Void>, FluentRestAction<Void, PremiumRequiredCallbackAction>
```

Comment:
This should return the InteractionHook like ReplyCallbackAction

Reply:
You can't further reply to this kind of interaction, it's kinda like modal interactions, does it still need to return `InteractionHook` in that case

Reply:
Can you not even delete it?

Reply:
No don't think so

Reply:
Fine by me.